build.gradle

```diff
@@ -23,7 +27,27 @@ repositories {
 }
 
 dependencies {
+    api 'javax.servlet:javax.servlet-api:3.1.0'
     testImplementation 'junit:junit:4.12'
+    testImplementation 'pl.pragmatists:JUnitParams:1.1.1'
+}
+
+tasks.withType(FindBugs) {
+    reports {
+        xml.enabled false
+        html.enabled true
+    }
+}
+
+test {
+    // In the meantime I implement the Library API
+    exclude '**/CookiePotTest*'
+}
+
+compileTestJava.options.encoding = 'UTF-8'
```

Comment:
Because some test cases make use of UTF-8 characters and the wrong enconding will lead to false negatives.